src/HtmlExtension.php

```diff
@@ -108,4 +109,22 @@ function twig_html_classes(...$args): string
 
     return implode(' ', array_unique($classes));
 }
+
+function twig_html_attributes(array $attributes): string
+{
+    $output = '';
+
+    foreach ($attributes as $attribute => $value) {
+        if ($value === true) {
+            $output .= ' ' . $attribute;
+        } else if (\is_string($value) || \is_numeric($value)) {
+            $value = \htmlspecialchars($value, ENT_COMPAT | ENT_HTML5, 'UTF-8', false);
```

Comment:
What if the Twig environment is configured with a different charset than UTF-8 ?

Reply:
my intention was to set double_quoting to false.
Do you have a better solution?